src/parquet/column/writer.h

```diff
@@ -56,9 +57,26 @@ class PARQUET_EXPORT ColumnWriter {
 
  protected:
   virtual std::shared_ptr<Buffer> GetValuesBuffer() = 0;
+  /**
+   * Serializes Dictionary Page if enabled
+   */
```

Comment:
Comment style is inconsistent, since we have been using `//` we should probably stick to that for now (https://google.github.io/styleguide/cppguide.html#Comment_Style).